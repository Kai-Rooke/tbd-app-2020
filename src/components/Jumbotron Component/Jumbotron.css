.jumbotron
{
    
    padding-top: 4% !important;
    padding-bottom: 4% !important;
    background-color: #ffffff;
    position: absolute;
    width: 40%;
    height: 25%;
    left: 50%;
    top: 30%;
    -webkit-transform: translate(-50%, -50%);
    transform: translate(-50%, -50%);
}
.jumbotron h3{
    text-align: center;
    font-size: 18px;
    font-weight: bolder;
}
.jumbotron p{
    text-align: center;
    font-size: 16px;
}

@media screen and (max-width: 600px) {
    .jumbotron { 
        padding-top: 2% !important;
        background-color: #a1a1a1;
        position: absolute;
        width: 75%;
        height: 10%;
        left: 50%;
        top: 30%;
        -webkit-transform: translate(-50%, -50%);
        transform: translate(-50%, -50%);
    }
    .jumbotron h3{
        text-align: center;
        font-size: 12px;
        font-weight: bolder;
    }
    .jumbotron p{
        text-align: center;
        font-size: 10px;
    }

  }