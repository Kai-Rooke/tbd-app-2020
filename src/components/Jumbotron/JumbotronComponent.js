function JumbotronComponent()
{
    return(
            <h1>Jumbotron Test 1</h1>
    );
}

export default JumbotronComponent;