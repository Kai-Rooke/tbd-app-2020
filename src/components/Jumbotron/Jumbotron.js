import React from 'react';
import './Jumbotron.css';

export const Jumbotron = ({ ...props }) => {
    return (
        <div className="jumbo" data-testid="jumbotron">
            {props.children}
        </div>        
    )
}

export default Jumbotron;