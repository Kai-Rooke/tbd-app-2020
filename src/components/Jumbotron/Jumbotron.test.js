import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Jumbotron from './Jumbotron';

//cleans up 
afterEach(cleanup);

it("Renders Jumbotron", () => {
    const { asFragment } = render(<Jumbotron className="jumbo"/>);
    expect(asFragment()).toMatchSnapshot();
});