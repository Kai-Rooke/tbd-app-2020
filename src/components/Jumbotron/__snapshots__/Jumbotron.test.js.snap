// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Jumbotron 1`] = `
<DocumentFragment>
  <div
    class="jumbo"
    data-testid="jumbotron"
  />
</DocumentFragment>
`;
